src/compiler/transformers/declarations.ts

```diff
@@ -579,7 +590,7 @@ export function transformDeclarations(context: TransformationContext) {
 
         function mapReferencesIntoArray(references: FileReference[], outputFilePath: string): (file: SourceFile) => void {
             return file => {
-                if (exportedModulesFromDeclarationEmit?.includes(file.symbol)) {
+                if (exportedModulesFromDeclarationEmit?.has(file.symbol)) {

```

Comment:
Does this not prevent inclusion of manually-written triple-slash references that are redundant with imports?